### ISSUE TITLE
Redirect after signup and redirect after login not functioning

### ISSUE THREAD

**Issue by ajaymenonv** [moxi9/phpfox#1162]
hi...i tried redirect option but seems some bug here - URL Redirect After Login , i mentioned it to redirect to invite and Redirect After SignUp to user/privacy ... but it is not working and it is using default routine, I have disabled email verification for your clarification.. we are using version 4.2.2

community discussion thread link is: https://community.phpfox.com/forum/thread/54228/redirect-not-working/

**Comment by Data66PF** [comment]
I've checked and our code in the database for the setting "same page redirect after login" is enabled by default and there is no way to disable it. This setting used to conflict with the redirects for this bug report. However, I even tested to go in the db and disable the setting and it had no impact even after cache clearing. We need to add that setting back as site owners do like to set it and it should also allow us to then make these redirects work if we set that setting to False. 

I'll confirm this as the setting does not work as shown and the other setting is missing to control it further.
